security_monkey/datastore.py

```diff
@@ -62,16 +70,37 @@ class Account(db.Model):
     id = Column(Integer, primary_key=True)
     active = Column(Boolean())
     third_party = Column(Boolean())
-    name = Column(String(32))
+    name = Column(String(32), unique=True)
+    s3_name = Column(String(64)) # (deprecated-custom)
+    number = Column(String(12))  # (deprecated-identifier) Not stored as INT because of potential leading-zeros.
     notes = Column(String(256))
-    s3_name = Column(String(64))
-    number = Column(String(12))  # Not stored as INT because of potential leading-zeros.
+    identifier = Column(String(256))  # Unique id of the account, the number for AWS.
     items = relationship("Item", backref="account", cascade="all, delete, delete-orphan")
     issue_categories = relationship("AuditorSettings", backref="account")
-    role_name = Column(String(256))
+    role_name = Column(String(256)) # (deprecated-custom)
```

Comment:
@monkeysecurity we were planning to do that as a seperate task because this change is already rather large, and deprecating these fields would make it a much larger code change, and would require a DB migration for existing data along with a rollback. We could go ahead do that if you would prefer to roll it all into one PR.